hexnumber/hexnumber.py

```diff
@@ -0,0 +1,84 @@
+import sys
```

Comment:
Hello! 
 I`m here to help you with making your pr title looks better. Now it is: 
 added hexnumber 
 Mistakes: 
 Dude...it is not even in right format.